masonite/drivers/CacheDiskDriver.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+
+class CacheDiskDriver():
```

Comment:
Let's focus on expanding this cache driver now. all caching should have these methods:

* `store` -- stores the value inside the key
* `get` -- gets the value from the key
* `delete` -- deletes the value

Once we implement those methods which should have signatures like:

```
def store(self, key, value):

def get(self, key)

def delete(self, key)
```

Once those methods are created, test them to make sure they all work. So basically do the same type of assertions as the views (run the method and then `glob()` the file location to check the value is saved).

The idea here is that a lot of the logic of a view shouldn't be caching. it should be rendering views. So once we have tests for `Cache` then we can implement the `Cache` inside the view.

The cache should be used individually and not just for caching templates. We are just also going to use then to cache templates by passing them the `key` (the template name) and the `value` (the `self.rendered_template` value).

Again we should be able to use this completely outside caching templates like in a controller:

```python
def show(self, Cache):
    Cache.get('key') # returns the value
    Cache.store('key', 'value', '.txt') # stores the value inside `bootstrap/cache/key:888r8975445.txt`. 
```

Maybe pass in an extension here as the third parameter? it should default to `.txt` but inside the `View` class we can just pass `.html` 

You could either build this from scratch (should be simple) or you can look at a python library for this